helm/rootcause/Chart.yaml

```diff
@@ -1,15 +1,16 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
+apiVersion: v2
+description: A Rootcause service helm chart. Part of Hydrosphere.io
```

Comment:
A helm chart for the rootcause service which provides explanations of model outputs.